### PR TITLE
Install cloudflared via Cloudflare Package Repository

### PR DESCRIPTION
**Pull request by stumpylog**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description

This changes the base image to intsall cloudflared via the [Cloudflare Package Repository](https://pkg.cloudflare.com/index.html).  This apt repo provides the package for amd64, arm64 and armv7, removing the need for a seperate download script or adding a new arch as a workaround.

The `--insecure` option of curl is required as buster's ca-certificates are too old to validate the server.  It can be removed if bullseye or bookworm are upgraded to instead.

## Type of change

Please delete any options that are not relevant.

- Other

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

**Comment by chakflying** [comment]
Seems like a good opportunity to upgrade to Bullseye. Raspberry Pi OS has been on bullseye for more than a year now. The underlying issue with `uv_gettimeofday` has [documented solutions](https://github.com/nodejs/docker-node/issues/1746) so it should not be a consideration anymore.

**Comment by stumpylog** [comment]
It would be great to do that, though I'd prefer it in a separate PR (or go in before this so I can remove the `--insecure`).

I think alpine should update at the same time.

File: docker/debian-base.dockerfile [hunk 1]
```diff
@@ -8,21 +8,21 @@ WORKDIR /app
 # Install Curl
 # Install Apprise, add sqlite3 cli for debugging in the future, iputils-ping for ping, util-linux for setpriv
 # Stupid python3 and python3-pip actually install a lot of useless things into Debian, specify --no-install-recommends to skip them, make the base even smaller than alpine!
-RUN apt update && \
-    apt --yes --no-install-recommends install python3 python3-pip python3-cryptography python3-six python3-yaml python3-click python3-markdown python3-requests python3-requests-oauthlib \
-        sqlite3 iputils-ping util-linux dumb-init git && \
+RUN apt-get update && \
```
Comment (louislam) [inline docker/debian-base.dockerfile:11]: Any reason why changed to `apt-get`?

**Comment by louislam** [comment]
> Seems like a good opportunity to upgrade to Bullseye. Raspberry Pi OS has been on bullseye for more than a year now. The underlying issue with `uv_gettimeofday` has [documented solutions](https://github.com/nodejs/docker-node/issues/1746) so it should not be a consideration anymore.

Yes, it will be updated in 2.0.

See https://github.com/louislam/uptime-kuma/pull/2720

> Update to the latest version of Debian (as Raspberry users can't use it directly, will try to provide a guide to update libseccomp2: https://github.com/sdr-enthusiasts/Buster-Docker-Fixes)

File: docker/debian-base.dockerfile (hunk 1 above)
Comment (stumpylog) [reply docker/debian-base.dockerfile:11]: Using plain `apt` outputs a warning about the lack of a stable interface.  It's not meant to be used in scripts basically, but apt-get is stable and outputs no warning.

**Pull request merged**